source/shared/core_sqlsrv.h

```diff
@@ -1639,8 +1642,9 @@ enum SQLSRV_ERROR_CODES {
 
     SQLSRV_ERROR_ODBC,
     SQLSRV_ERROR_DRIVER_NOT_INSTALLED,
-    SQLSRV_ERROR_AE_DRIVER_NOT_INSTALLED,
+    SQLSRV_ERROR_AE_DRIVER_REQUIRED,
```

Comment:
multiple occurrances:
I know it's confusing, but it's actually CE. The flags, and all Always Encrypted docs talk about CE. It's SQL_CE_ENABLED in msodbcsql.h too it would be confusing to call talk about an AE driver here